scripts/test

```diff
@@ -26,5 +26,10 @@ if [[ "${BASH_SOURCE[0]}" == "$0" ]]; then
         # docker-compose \
         #     run --rm --no-deps \
         #     black --check --diff .
+
+        # build and test cog-clip assembly
+        # we don't need bloop, sbt uses .bsp
+        cd src/pipeline/activator/cog-clip/; ./sbt ";test; assembly"; cd ~- 
+
```

Comment:
I'd consider keeping the invocation of `./sbt test`, but moving the assembly into `cibuild` right before we build the container image.

Also, a nit, but we usually wrap blocks in `pushd` / `popd` blocks rather than using `cd`. e.g.

```bash
pushd ./src/pipeline/activator/cog-clip
./sbt test
popd
```

Reply:
@rbreslow tbh I don't see any benefits of splitting out the test + assembly build into two separate stages; looks to me like encrypting commands location for now: creates more ambiguity; I would see benefits if these were separate github action steps (would be easier to inspect the CI error)

Btw, it is a usual strategy to run tests before building the fat assembly jar.

Reply:
The test and assembly split is done.

Reply:
The purpose of `test` is to [consolidate all testing procedures](https://github.com/azavea/architecture/blob/master/doc/arch/adr-0000-scripts-to-rule-them-all.md). The purposes of `update` and `cibuild` are to build artifacts. I'd make the case that it's more ambiguous to overload the purpose of `test`—It'd end up doing things I wouldn't expect it to.